resources/views/welcome.blade.php

```diff
@@ -461,6 +472,7 @@ class="portfolio-item--content simple-ajax-popup">
                                 <button class="btn hover-state">Submit</button>
                             </div>
                         </div>
+                        <?php endif ?> 
```

Comment:
According to the convention,.
```suggestion
    @endif
```